src/frontend/utils/tidslinje.ts

```diff
@@ -21,48 +14,35 @@ export const splittYtelseVedEndringerPåAnnenYtelse = (
         (acc, periodeSomSplitterOpp) => {
             const sisteElement: Periode = acc[acc.length - 1];
 
-            const småFom = periodeSomSplitterOpp.stønadFom;
-            const småTom = periodeSomSplitterOpp.stønadTom;
+            const småFom = parseIsoString(periodeSomSplitterOpp.stønadFom);
```

Comment:
Litt usikker på navngivningen her, endrer gjerne men kommer ikke på et passende navn i farten

Reply:
Hva står `små` for? 

Kunne `fomSplitt` funka?

Reply:
Ser ut som det er småbarnstillegg som alltid er det som splitter opp, så eventuelt fomSmåbarnstillegg, men da burde man endre inputparameteren til funksjonen.

Reply:
Kan også være utvidet hvis det er institusjon hvis jeg husker riktig 😅 så tror jeg må gå for en generisk variant